crates/red_knot_python_semantic/src/types/display.rs

```diff
@@ -14,7 +14,7 @@ impl<'db> Type<'db> {
     pub fn display(&self, db: &'db dyn Db) -> DisplayType {
         DisplayType { ty: self, db }
     }
-    fn representation(self, db: &'db dyn Db) -> DisplayRepresentation<'db> {
+    pub(super) fn representation(self, db: &'db dyn Db) -> DisplayRepresentation<'db> {
```

Comment:
I think we can make this private again? It really shouldn't be used outside this module.